dev-qt/qtbase/qtbase-6.7.2-r1.ebuild

```diff
@@ -12,7 +12,7 @@ if [[ ${QT6_BUILD_TYPE} == release ]]; then
 fi
 
 declare -A QT6_IUSE=(
-	[global]="+ssl +udev zstd"
+	[global]="+ssl +udev zstd systemd syslog"
```

Comment:
Does it build fine if journald and syslog are enabled at same time? (just wondering if it conflicts, if it does may need a required use entry, otherwise it can be ignored)

Reply:
Yes, but it seems only the journald one will be available (https://github.com/qt/qtbase/blob/b57fac818b7fe028dc61f1196cfda397233edffe/src/corelib/global/qlogging.cpp#L1983). I'll add a required use entry to make it clear.